mllib/src/main/scala/org/apache/spark/ml/tree/treeModels.scala

```diff
@@ -58,3 +58,43 @@ trait DecisionTreeModel {
     header + rootNode.subtreeToString(2)
   }
 }
+
+/**
+ * :: AlphaComponent ::
+ *
+ * Abstraction for models which are ensembles of decision trees
+ *
+ * TODO: Add support for predicting probabilities and raw predictions
+ */
+@AlphaComponent
+trait TreeEnsembleModel {
```

Comment:
Yeah, I think I opened it up in order to write examples which worked for both classification and regression.  I'll change those.